app/routes/user/endorsements.server.ts

```diff
@@ -205,38 +206,52 @@ export class EndorsementsServer {
 
     let promiseArray: Promise<void>[] = []
 
-    if (status === 'approved')
+    let requestorMessage = `You are receiving this email because the status of your peer endorsment requested from ${
+      this.#currentUserEmail
+    } has been updated to ${status}.`
+
+    if (status === 'approved') {
       promiseArray.push(
         this.#addUserToGroup(requestorSub),
         clearUserToken(requestorSub)
       )
-
-    if (status === 'reported')
+      requestorMessage += `
+      
+      As an approved user, you may now submit GCN Circulars at ${origin}/circulars/new and be requested for endorsement by other users.
+      
+      View your pending endorsements here: ${origin}/user/endorsements`
```

Comment:
This last line of the text with the link to the endorsements page should be consistent across different emails too.

Reply:
Are you thinking `View your pending endorsements here: ${origin}/user/endorsements` for all of them, or something that sounds like it applies to either endorser or requester?

Reply:
I think it applies to either.